### PR TITLE
scan interface

### PR DESCRIPTION
**Pull request by critical27**
As title, the interface is almost same as 1.0.

### CODE REVIEW

File: src/common/interface/storage.thrift [hunk 1]
```diff
@@ -607,6 +607,66 @@ struct LookupAndTraverseRequest {
  * End of Index section
  */
 
+struct ScanVertexRequest {
+    1: common.GraphSpaceID                  space_id,
+    2: common.PartitionID                   part_id,
+    // start key of this block
+    3: optional binary                      cursor,
+    4: list<VertexProp>                     return_columns,
```
Comment (darionyaphet) [inline src/common/interface/storage.thrift:615]: If so, you can support multi tag when running scan once?

**Review by bright-starry-sky — APPROVED** [review]
LGTM.

File: src/common/interface/storage.thrift (hunk 1 above)
Comment (critical27) [reply src/common/interface/storage.thrift:615]: Yeah, so does 1.0

Comment (darionyaphet) [reply src/common/interface/storage.thrift:615]: Maybe only support one tag with multi column is enough. 

For Spark someone fetch multi tags into one frame.  IMO, this interface maybe make the user confuse.

Comment (critical27) [reply src/common/interface/storage.thrift:615]: Yeah, we could just add only one tag in the list in that case.

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**